src/views/assets/Cloud/Account/components/AuthPanel.vue

```diff
@@ -175,18 +175,16 @@ export default {
       form.validate((valid) => {
         if (valid) {
           btn.loading = true
+          this.$refs.form.$refs.form.dataForm.submitForm('form', false)
+          if (this.origin === 'update') {
+            setTimeout(() => {
+              this.$emit('submitSuccess')
+              this.$emit('update:visible', false)
+            }, 500)
+          }
+          this.submitType = submitType
         }
       })
-
-      this.$refs.form.$refs.form.dataForm.submitForm('form', false)
-
-      if (this.origin === 'update') {
-        setTimeout(() => {
-          this.$emit('submitSuccess')
-          this.$emit('update:visible', false)
-        }, 500)
-      }
-      this.submitType = submitType
     },
     handleSubmitSuccess(res) {
       if (this.submitType === 'manual') {
```

Comment:
None Found

There was no need to conduct any checks or adjustments for this code snippet from 2021-09-01 compared to the current date of 2025-04-15.

If you have other codes that require checking and suggestions, please provide them so I can assist further in analyzing their structure and functionality.